README.md

```diff
@@ -1,13 +1,45 @@
-# RdTools
+# About RdTools
 
-Degradation Tools
+RdTools is a set of Python tools for analysis of photovoltaic data.
+In particular, high frequency (hourly or greater) PV production data is evaluated
+over several years to obtain rates of performance degradation over time.
 
-Python tools for photovoltaic degradation analysis.
+## Workflow
 
-## Install using pip
+0. Import and preliminary calculations
+1. Normalize data using a performance metric
+2. Filter data that creates bias
+3. Aggregate data
+4. Analyze aggregated data to estimate the degradation rate
+
+
+<img src="./screenshots/Workflow1.png" width="600" height="331" alt="RdTools Workflow"/>
+
+## Degradation Results
+
+RdTools output shows filtered historical data along with a linear trend line.
+However, the most accurate estimate of degradation over the analysis period in question
+is the Year-on-year (YOY) distribution, which is a normal distribution, the central tendancy
+of which is the most representative degradation rate.  Distribution width provides information
+about the accuracy of the analysis.
+
+<img src="./screenshots/Clearsky_result.png" width="600" height="456" alt="RdTools Result"/>
+
+
+Two methods are available for system performance ratio calculation.  The sensor-based
```

Comment:
I suggest changing this to say "two workflows are available" to avoid confusion (since method has a technical meaning in python).  It would also be worth noting here that both are illustrated in the example notebook.

Reply:
OK